azerothcore/azerothcore-wotlk
# fix(Scripts/Zone): Move zangarmarsh npcs to SAI

Currently Kayra won't heal anyone because of https://github.com/azerothcore/azerothcore-wotlk/issues/15515
## Changes Proposed:
- Move zangarmarsh npcs to SAI

## Issues Addressed:

- Closes https://github.com/azerothcore/azerothcore-wotlk/issues/4147
- Closes https://github.com/chromiecraft/chromiecraft/issues/4751
- Closes https://github.com/azerothcore/azerothcore-wotlk/issues/14692
- Progress https://github.com/azerothcore/azerothcore-wotlk/issues/15024 https://github.com/azerothcore/azerothcore-wotlk/issues/13898 https://github.com/chromiecraft/chromiecraft/issues/4969

## SOURCE:

Co-authored-by: aletson <aletson@users.noreply.github.com>
 https://github.com/TrinityCore/TrinityCore/pull/20773
 https://github.com/TrinityCore/TrinityCore/pull/20430
 
Co-authored-by: tkrokli  <tkrokli@users.noreply.github.com> (deleted)
https://github.com/TrinityCore/TrinityCore/commit/0c6eeb3abc54dbc0f147e5098619e8e19e27accd
https://github.com/TrinityCore/TrinityCore/commit/e2f395863a7fb1a85dc7d09f2692198faae16f46

Co-authored-by: Faq <Faq@users.noreply.github.com>
Co-authored-by: offl <offl@users.noreply.github.com> 
https://github.com/Faq/TrinityCore/commit/2219319d5a5a3e04e81548d23efaa177879bbb29
## Tests Performed:

- tested


## How to Test the Changes:


Coosh'coosh (18586)
.go c id 18586
Check for gossip, should be none
.q a 10009
Check for gossip, should have "I have been sent by ..."
Check abilities, should have Lighting shield (10s cd), and should spam Lighting Bolts (until oom)
On After Combat, On Respawn or Spawn should go back to being friendly.

Mortog Steamhead (23373)
.go c id 23373
Check gossip, should be none
.modify rep 942 friendly/honored/revered - no gossip
.modify rep 942 exalted - shold have vendor gossip

(Alliance)
Timothy Daniels (18019)
.go c id 18019
Check gossip options
1) Vendor
2) "Specialist, eh?..." -> "If you have to ask..."

Elder Kuruti (18197)
.go c id 18197
.q a 9803
Use the item
Check for gossip, should be "Greetings, elder ..." -> "I did not mean to ..." -> "I will tell them ..." -> Receive the quest item

Kayra Longmane(17969)
.go c id 17969
Do the quest, check text, combat, waypoints, summon groups, fail/complete status
Check if she heals players below 25% health, within 25y. **(CURRENTLY NOT WORKING FROM CORE)**
## Known Issues and TODO List:


- [ ] Kayra Longmane healing touch timer/health percent needs to be checked on live (or video source, couldn't find any), current ones are assumed.

## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.
